.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
Make sure to remove `.idea` from git. Adding an entry to the `.gitignore` will not remove the files from git if they have already been committed.